### DISCUSSION TITLE
Support for file attachments

### DISCUSSION THREAD

**Discussion by starryxoxo** [alangrainger/share-note#143] [Vote for upcoming features]
I suppose the images work (do they?) but not audio attachments. Can audio attachments be supported soon? Or is it supported already and I'm just not finding the right way of attaching the file?

**Comment by alangrainger** [comment]
At the moment this is not planned.

Share Note is for sharing your notes; it’s not a file-sharing service.

The correct way to share any large file is to upload it to a proper file sharing service, and then link the shared file back into your note.

https://docs.note.sx/faq

**Comment by alangrainger** [comment]
This may be added in the future - follow discussion #119 for updates
